CHANGELOG.md

```diff
@@ -484,7 +484,8 @@ Released with 1.0.0-beta.37 code base.
 
 ### Added
 - `maxPriorityFeePerGas` and `maxFeePerGas` added to `Transaction` and `TransactionConfig` interfaces (#4232) (#4585)
-
+- `transactionPollingInterval` added to web3, contract and method constructor options. defaults to 1 second. (#4584)
```

Comment:
1.7.0 is already frozen under 1.7.0-rc this change should be under future releases